.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,46 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
+<summary><strong>Guidelines</strong></summary>
+Thank you for making a feature suggestion! :heart:
 
-<!-- Please fill these sections with the relevant information: -->
+Please fill the sections below with the relevant information.
 
-# A detailed description of the proposed feature
+Homebrew is a non-profit project run entirely by unpaid volunteers.
+We, therefore, reserve the right to close your issue without comment if you
+delete, do not read, or do not fill out the issue checklist below and provide
+**all** the requested information. Note, that if you repeatedly fail to use
+the issue template, we will be forced to block you from ever submitting issues
+to Homebrew again.
 
-<!-- replace me -->
+We will also likely close this issue or ask you to create a pull-request if
+it's something the maintainers are not actively planning to work on.
 
-# The motivation for the feature
+Thank you for your understanding,<br>
+\- Homebrew maintainers
+</details>
 
-<!-- replace me -->
 
-# How the feature would be relevant to at least 90% of Homebrew users
```

Comment:
I’d say this point is too important to be relegated to a hidden comment. When the conversation on a feature evolves, there should always be a return to “but is that relevant to the majority of users?”. Having that front and centre is an important reminder and makes the goal clear even to commenters who never opened a feature request.

“Relevance” is generic and open to interpretation; “How the feature would be relevant to at least 90% of Homebrew users” is specific and direct. From anecdotal experience on the cask side, it a note is a comment instead of direct text, people are considerably more likely to not follow it.

In general, I think the template as it stands is isn’t cleaner than your changes, but it’s clearer and that is more important.

Reply:
This text is found nearly verbatim below on line 44.

Reply:
> This text is found nearly verbatim below on line 44.

It is. My sole comment was that it was hidden. Not sure if that’s a problem on this repo as well, but *anecdotally* on the Cask side hidden comments tended to be ignored by users a lot.

Reply:
This is/should be more prominent and first to discourage people typing a detailed description only to realise "oh, this will be useful to almost no-one so I shouldn't bother submitting this".